docs/docs/integrations/memory/cratedb_chat_message_history.ipynb

```diff
@@ -0,0 +1,357 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# CrateDB Chat Message History\n",
+    "\n",
+    "This notebook demonstrates how to use the `CrateDBChatMessageHistory`\n",
+    "to manage chat history in CrateDB, for supporting conversational memory."
+   ],
+   "metadata": {
+    "collapsed": false
+   },
+   "id": "f22eab3f84cbeb37"
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Prerequisites"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "!#pip install 'langchain[cratedb]'"
```

Comment:
That's no longer a valid installation incantation. Something like this would be better:

```suggestion
    "!#pip install langchain sqlalchemy-cratedb"
```